services/jupyterhub/spawners.py

```diff
@@ -286,6 +304,8 @@ def get_pod_manifest(self):
                 image='alpine/git',
                 command=['sh', '-c'],
                 args=[
+                    'rm -rf {mount_path}/* && '
+                    '(rm -rf {mount_path}/.* || true) && '
```

Comment:
this can fail?

Reply:
Well sort of, it returns 1 with "cannot remove  . and .."